src/jit/simd.cpp

```diff
@@ -754,7 +754,7 @@ SIMDIntrinsicID Compiler::impSIMDLongRelOpEqual(CORINFO_CLASS_HANDLE typeHnd,
     //
     // Equality(v1, v2):
     // tmp = (v1 == v2) i.e. compare for equality as if v1 and v2 are vector<int>
-    // result = BitwiseAnd(t, shuffle(t, (2, 3, 1 0)))
+    // result = BitwiseAnd(t, shuffle(t, (2, 3, 0, 1)))
     // Shuffle is meant to swap the comparison results of low-32-bits and high 32-bits of respective long elements.
```

Comment:
Note that here, and in the code below, I have simply changed the comment and the name of the shuffle constant to correctly reflect the actual shuffle that was (already) being done. I discovered this as I was changing the dot product code to use shuffle constants to make it easier to follow what is being done.